.circleci/config.yml

```diff
@@ -599,6 +599,21 @@ jobs:
             - solana-programs/claimable-tokens/program/target
           key: claimable-token-deps-{{ checksum "solana-programs/claimable-tokens/program/Cargo.toml" }}
 
+      - run:
+          name: Setup Anchor
+          command: |
+            sudo apt-get update
+            sudo apt-get install pkg-config libssl-dev libudev-dev
+            cargo install --git https://github.com/project-serum/anchor --tag v0.22.1 anchor-cli --locked
```

Comment:
does it slow things down if the build already installs these? added you to my PR https://github.com/AudiusProject/audius-protocol/pull/2691

Reply:
I cached this, thanks